pkg/operator/ceph/csi/spec.go

```diff
@@ -367,6 +367,44 @@ func startDrivers(namespace string, clientset kubernetes.Interface, ver *version
 			return errors.Wrap(err, "failed to load cephfs plugin service template")
 		}
 	}
+
+	pv, _ := clientset.CoreV1().PersistentVolumes().List(metav1.ListOptions{})
+	drivers := []string{CephFSDriverName, RBDDriverName}
+	driversPV := false
+	for _, p := range pv.Items {
```

Comment:
As mentioned in [this comment](https://github.com/rook/rook/pull/5605#pullrequestreview-428286788), I would suggest we still honor the desired state and stop the driver. It's going to be rarely changed anyway so I don't think we need to worry about the corner case of accidentally disabling the driver. @Madhu-1 sound reasonable?

Reply:
So we should just disable the drivers if it is set to false?

Reply:
one problem with that is, we can leave stale rbd maps and cephfs mounts if we disable the CSI driver if the PVC is in use. we need to test below scenario and make sure delete PVC works when we enable it back

* Create PVC and Bind it to an app
* Disable CSI driver
* Delete App
* Enable CSI driver
* Delete PVC.
* Verify there is no stale rbd mapped images and stale cephfs mounts in plugin pod
if above case works we should be fine.

Reply:
@subhamkrai After you confirm what happens according to Madhu's manual test steps, if the rbd volume can be unmounted after reenabling the csi driver, then there is no need for the PV check. In that case, you can remove lines 371-387 and just directly remove the csi drivers.

Reply:
I tested Madhu's manual test steps, as PV is getting deleted so, we don't need to check for PV.

Reply:
Not check pvc might create issue when we add support for userspace clients. Need more discussion
Cc @ShyamsundarR

Reply:
@Madhu-1 How will the userspace clients make a difference? Since CSI will retry removing the volume until the CSI driver responds, reinstalling the CSI driver sounds like a confirmed workaround. Anyway, this scenario of disabling the CSI driver would be very rare when there are existing PVs so I'd like to keep it simple and avoid the PV checks.

Reply:
If the PVC is mounted using a fuse client, we will see Transport Endpoint not connected error, this won't block the Pod deletion, only disadvantage if csi-driver restarts we cannot use PVC in app pod, its a known issue for fuse clients now. i think we are good, we can just disable CSI driver without checking PV or PVC

Reply:
Good then we don't need to check for PVs. Changes done.